packages/node/src/integrations/http.ts

```diff
@@ -275,3 +279,11 @@ function getBreadcrumbData(request: ClientRequest): Partial<SanitizedRequestData
 function _isClientRequest(req: ClientRequest | HTTPModuleRequestIncomingMessage): req is ClientRequest {
   return 'outputData' in req && 'outputSize' in req && !('client' in req) && !('statusCode' in req);
 }
+
+/**
+ * Detects if an incoming request is a prefetch request.
+ */
+function isKnownPrefetchRequest(req: HTTPModuleRequestIncomingMessage): boolean {
+  // Currently only handles Next.js prefetch requests but may check other frameworks in the future.
+  return req.headers['next-router-prefetch'] === '1';
```

Comment:
Can we look at the `Sec-Purpose` header instead? This should work with all header types.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-Purpose

Reply:
Oh wait it's not standardized 😭 https://chromestatus.com/feature/6247959677108224